CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [[v9.?.?](https://github.com/terraform-aws-modules/terraform-aws-eks/compare/v9.0.0...HEAD)] - 2020-xx-xx]
 
+- Fix #753 by adding and instance tag spec to launch templates (by @js-timbirkett)
```

Comment:
Can you please your changes in changelog ? It's now generated from git commits during release.